components/Header.js

```diff
@@ -1,23 +1,111 @@
 import styled from "styled-components";
+import Link from "next/link";
 
 const NavContainer = styled.header`
   display: flex;
+  position: fixed;
   justify-content: center;
   align-items: center;
+  top: 0px;
   height: 60px;
```

Comment:
Just a general note, not just for this but for everything: You might want to think about using relative units (like rem etc.) so that the sizes can adjust